web/src/features/fireBehaviourCalculator/components/FBCInputGrid.tsx

```diff
@@ -74,106 +121,268 @@ const FBCInputGrid = (props: FBCInputGridProps) => {
+        <TableContainer className={classes.tableContainer}>
+          <Table stickyHeader aria-label="daily table view of HFI by planning area">
+            <TableHead>
+              <TableRow>
+                <TableCell>
+                  <Checkbox
+                    color="primary"
+                    checked={headerSelected}
+                    onClick={() => {
+                      if (headerSelected) {
+                        // Toggle off
+                        updateSelected([])
+                        setHeaderSelect(false)
+                      } else {
+                        updateSelected(inputRows.map((_, i) => i))
+                        setHeaderSelect(true)
+                      }
+                    }}
+                  />
+                </TableCell>
+                <TableCell key="header-location">Weather Station</TableCell>
+                <TableCell key="header-elevation">
+                  Elev.
+                  <br />
+                  (m)
+                </TableCell>
+                <TableCell key="header-fuel-type">
+                  FBP
+                  <br />
+                  Fuel
+                  <br />
+                  Type
+                </TableCell>
+                <TableCell>
+                  Grass
+                  <br />
+                  Cure
+                  <br />
+                  (%)
+                </TableCell>
+                <TableCell>Status</TableCell>
+                <TableCell>
+                  Temp
+                  <br />
+                  (&deg;C)
+                </TableCell>
+                <TableCell>
+                  RH
+                  <br />
+                  (%)
+                </TableCell>
+                <TableCell>
+                  Wind
+                  <br />
+                  Dir
+                  <br />
+                  (&deg;)
+                </TableCell>
+                <TableCell className={classes.windSpeed}>
+                  {'Wind Speed (km/h) (Optional)'}
+                  <Tooltip title="Leave this empty to calculate forecasted/observed wind speed. Add a custom wind speed to influence the calculations">
+                    <InfoIcon aria-label="info"></InfoIcon>
+                  </Tooltip>
+                </TableCell>
+                <TableCell>
+                  Precip
+                  <br />
+                  (mm)
+                </TableCell>
+                <TableCell>FFMC</TableCell>
+                <TableCell>DMC</TableCell>
+                <TableCell>DC</TableCell>
+                <TableCell>ISI</TableCell>
+                <TableCell>BUI</TableCell>
+                <TableCell>FWI</TableCell>
+                <TableCell>HFI</TableCell>
+                <TableCell className={classes.criticalHoursCell}>
+                  Critical
+                  <br />
+                  Hours
+                  <br />
+                  (4000 kW/m) &dagger;
```

Comment:
we have &dagger; s in the column labels but we don't have the corresponding disclaimer box about critical hour calculations anymore. (Was it intentional to remove that disclaimer? If so, we should also delete the &dagger; s)

Reply:
Oops yeah looks like prod has removed the disclaimers altogether, I'll remove the daggers.

Reply:
well let's check with @tessmgood - was there an agreement made yesterday that the disclaimer box for critical hours is no longer required?